Formula/kibana.rb

```diff
@@ -45,8 +47,10 @@ def install
     # do not build zip package
     inreplace buildpath/"tasks/build/archives.js", /(await exec\('zip'.*)/, "// \\1"
 
+    # set npm env and fix cache edge case (https://github.com/Homebrew/brew/pull/37#issuecomment-208840366)
     ENV.prepend_path "PATH", prefix/"libexec/node/bin"
-    system "npm", "install"
+    Pathname.new("#{ENV["HOME"]}/.npmrc").write Language::Node.npm_cache_config
```

Comment:
Oh, I thought the latter would work here but apparently not in this case. It's fine as-is then.